src/libsystemd-network/dhcp-server-lease-internal.h

```diff
@@ -20,15 +20,15 @@ typedef struct sd_dhcp_server_lease {
         be32_t address;
         be32_t gateway;
         uint8_t chaddr[16];
-        usec_t expiration;
+        triple_timestamp expiration;
```

Comment:
hmm, i think we should only keep the timestamp in CLOCK_BOOTTIME in memory, but when writing it out convert it to CLOCK_REALTIME and write it out too in that format then.

Why? simply because dhcp expiration timeouts can be super long, and systems sometimes come up with bad clocks. hence we should only convert CLOCK_BOOTTIME to CLOCK_REALTIME at the moment we really have to, i.e. when writing, but never before. If that makes any sense?